contracts/TreasuryModule.sol

```diff
@@ -19,6 +19,7 @@ contract TreasuryModule is ERC721Holder, ModuleBase, ITreasuryModule {
         address _accessControl
     ) external initializer {
         __initBase(_accessControl, msg.sender, "Treasury Module");
+        _registerInterface(type(ITreasuryModule).interfaceId);
```

Comment:
Is there a reason why the overridden `supportsInterface()` wasn't removed from this contract? Does it have to do with `ERC721Holder`?

Reply:
Yes, this should have been removed. ERC721Holder doesn't have any ERC-165 support. Removed this in the latest commit.

I guess this brings up the larger question that has come up in the Governor repo. Do we want all interfaces inherited from abstract contracts to be ERC-165 supported?

Reply:
Idk, what are the pros and cons for that decision?

Reply:
The pro is that all possible interfaces could be recognized through ERC-165. 

The main con I think is trying to support all possible external functions in a contract through ERC-165 could get unwieldy, especially when thinking about the Governor module repo. 

I think it comes down to what is our intent when implementing ERC-165. I think of it as primarily being a way for a frontend to know what type of module it is dealing with. For that use-case, it feels like we probably just mainly care about the top level contract interface.